### PR TITLE
feat/merge serverless-py

### PR DESCRIPTION
**Pull request by mkandan**
see [readme](https://github.com/mkandan/dubdubs/blob/serverless-py/gcp-cloud-functions/readme.md)

### CODE REVIEW

**Review by Olyno — CHANGES_REQUESTED** [review]
Good work! I still have some questions about the pr

File: gcp-cloud-functions/.env.example
[the remark this answers is not in the document]
Comment (mkandan) [reply gcp-cloud-functions/.env.example:1]: this just an example of whats needed by gcp, has no values for variables

File: gcp-cloud-functions/translate-en/convert_to_iso_639_1.py
[the remark this answers is not in the document]
Comment (mkandan) [reply gcp-cloud-functions/translate-en/convert_to_iso_639_1.py:1]: needed for each cloud function (deployed separately). couldn't find a better way to "share code" between cloud functions without hosting code bases on google via their "artifact registry" or other code repo services

File: gcp-cloud-functions/translate-not-en/requirements.txt
[the remark this answers is not in the document]
Comment (mkandan) [reply gcp-cloud-functions/translate-not-en/requirements.txt:1]: separate requirements for separate cloud functions. 

> needed for each cloud function (deployed separately). couldn't find a better way to "share code" between cloud functions without hosting code bases on google via their "artifact registry" or other code repo services

**Review by mkandan — COMMENTED** [review]
applicable changes made

**Review by mkandan — COMMENTED** [review]
done? forgot to click viewed on one?

**Review by Olyno — APPROVED** [review]
LGTM!

**Pull request merged**